src/Symfony/Component/IntlEmoji/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2022-present Fabien Potencier
```

Comment:
```suggestion
Copyright (c) 2023-present Fabien Potencier
```

the new component is, if merged introduced in 2023 or 2024

Reply:
Ok i did not know, as the code is "moved". Fixed.